Makefile

```diff
@@ -31,9 +31,9 @@ test: unit-tests build-examples
 unit-tests: $(tests_binary)
 	$^ --exclude=integration --sequential
 
-$(tests_binary): $(SOURCE_FILES) | $(BUILD_DIR)
+$(tests_binary): $(wildcard test/*.pony) $(SOURCE_FILES) | $(BUILD_DIR)
```

Comment:
Given this is a library, we could move the test related things into peg/test and keep this the same as the other libraries.

Reply:
I've reverted the Makefile changes and placed the tests in `peg/_test.pony`